### PR TITLE
README: Add slack channel for community activity

### PR DESCRIPTION
**Pull request by ailun258**
Fixes: #5264

### CODE REVIEW

**Comment by ailun258** [comment]
@likebreath @liuw  please help to check again,

File: README.md [hunk 1]
```diff
@@ -373,7 +373,7 @@ are all equal and welcome means of contribution. See the
 ## Slack
 
 Get an [invite to our Slack channel](https://join.slack.com/t/cloud-hypervisor/shared_invite/enQtNjY3MTE3MDkwNDQ4LWQ1MTA1ZDVmODkwMWQ1MTRhYzk4ZGNlN2UwNTI3ZmFlODU0OTcwOWZjMTkwZDExYWE3YjFmNzgzY2FmNDAyMjI)
-and [join us on Slack](https://cloud-hypervisor.slack.com/).
+, and [join us on Slack](https://cloud-hypervisor.slack.com/), and [participate our community activities](https://cloud-hypervisor.slack.com/archives/C04R5DUQVBN).
```
Comment (liuw) [inline README.md:376]: Please drop the first `and`.

Please change `participate` to `participate in`.

Thanks.

Comment (ailun258) [reply README.md]: done.

**Review by likebreath — COMMENTED** [review]
Please squash the two commits.

**Review by likebreath — APPROVED** [review]
@ailun258 I squashed the commits on your behalf. The PR now is ready to be landed.

**Pull request merged**